service/start.go

```diff
@@ -46,10 +46,11 @@ func (svc *service) startNostr(ctx context.Context, encryptionKey string) error
 		defer svc.wg.Done()
 		//Start infinite loop which will be only broken by canceling ctx (SIGINT)
 		var relay *nostr.Relay
+		immediateRetry := false
```

Comment:
I think it'd be easier to understand to flip this and call it `waitToReconnectSeconds`, and use a number instead of a boolean, so that later we are able to easily adjust the algorithm (e.g. do an exponential back off algorithm)